packages/stems/src/utils/scrollParent.ts

```diff
@@ -19,7 +19,9 @@ export const getScrollParent = (node: HTMLElement): Element | null => {
     style(_node, 'overflow') +
     style(_node, 'overflow-y') +
     style(_node, 'overflow-x')
-  const scroll = (_node: HTMLElement) => regex.test(overflow(_node))
+  const scroll = (_node: HTMLElement) =>
+    regex.test(overflow(_node)) ||
+    _node.classList.contains('scrollbar-container') // Perfect scrollbar does overflow: hidden
```

Comment:
without this, we weren't detecting scroll parents that were using `react-perfect-scrollbar`